www/MobileServices.Web.js

```diff
@@ -10646,11 +10645,14 @@
 			},
 			gcm = function (push) {
 			    this._push = push;
+			},
```

Comment:
There's an error on this line.
The correct would be:
         wns = function (push) {
			    this._push = push;
            }